lib/internal/fs/utils.js

```diff
@@ -215,6 +216,15 @@ for (const name of ReflectOwnKeys(Dirent.prototype)) {
   };
 }
 
+ObjectDefineProperty(Dirent.prototype, 'path', {
+  __proto__: null,
+  get: deprecate(function() {
+    return this.parentPath;
+  }, 'dirent.path is deprecated in favor of dirent.parentName', 'DEP0178'),
```

Comment:
I believe you need to also add a setter for this, or it would be a breaking change.

Reply:
It is a breaking change, it's semver-major

Reply:
But this change also removes the setter and it is not documented.

Reply:
Thank you